### DISCUSSION TITLE
[Feature] Link to timeline from image menu

### DISCUSSION THREAD

**Discussion by ennob** [immich-app/immich#10434] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When viewing an image in my library that I found, e.g. by searching for something, I often find myself going to the main timeline to see other images from that day. It would be great to have a link in the image-menu that would take me directly there, rather than having to look at the date and manually find it.

In short, could we add a "go to image on main timeline" link to the image menu?

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#4115
